src/App.css

```diff
@@ -13,12 +13,21 @@
   --gray-500: rgb(128, 128, 128);
   --gray-900: rgb(169, 169, 169);
   --blue-600: rgb(0, 0, 255);
+  --black-900: rgb(0, 0, 0);
   --txt-margin: 2rem 0 .5rem 0;
   --txt-font-size: 1rem;
   --red-900: rgb(255, 0, 0);
   --main-body: rgb(247, 240, 240);
 }
 
+  .App {
+    background-color: rgb(244, 246, 247);
+    /* display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    height: 100vh; */
+  }
+
 button {
   cursor: pointer;
 }
```

Comment:
Commented code can be removed, also, do not target the elements directly, instead use class or id